source/agora/consensus/EnrollmentManager.d

```diff
@@ -118,8 +118,17 @@ public class EnrollmentManager
     /// Enrollment pool managing enrollments waiting to be a validator
     private EnrollmentPool enroll_pool;
 
-    /// The number of cycles for a bulk of pre-images
-    private immutable uint NumberOfCycles = 100;
+    /***************************************************************************
+
+        The number of enrollments we can emit in a cycle
+
+        With the current setting of ValidatorCycle being 1008 and blocks
+        having 10 minutes of interval, this gives us roughly 700 days,
+        or two years, per cycle.
+
+    ***************************************************************************/
+
+    private immutable uint EnrollPerCycle = 100;
```

Comment:
I'm not sure this name is that much better. The name seems to suggest there will be 100 enrollments per cycle, and a cycle is 1008 blocks? But that doesn't make sense.

Reply:
A cycle is 100 enrollments, and enrollment is 1008 blocks.

Reply:
We use a cycle in white paper as the period where a node is validator. So how about `CyclesPerNonce`?